content/telegraf/v1/plugins.md

```diff
@@ -9,61 +9,11 @@ menu:
     weight: 10
 weight: 6
 aliases:
-  - /telegraf/v1/plugins/processors/
-  - /telegraf/v1/plugins/plugins-list/
-  - /telegraf/v1/plugins/plugin-list/
-  - /telegraf/v1/plugins/aggregators/
-  - /telegraf/v1/plugins/outputs/
-  - /telegraf/v1/plugins/inputs/
-  - /telegraf/v1/plugins/processors/
-  - /telegraf/v1/plugins/plugins-list/
-  - /telegraf/v1/plugins/aggregators/
-  - /telegraf/v1/plugins/outputs/
-  - /telegraf/v1/plugins/inputs/
-  - /telegraf/v1/plugins/processors/
-  - /telegraf/v1/plugins/plugins-list/
-  - /telegraf/v1/plugins/aggregators/
-  - /telegraf/v1/plugins/outputs/
-  - /telegraf/v1/plugins/inputs/
-  - /telegraf/v1/plugins/processors/
-  - /telegraf/v1/plugins/plugins-list/
-  - /telegraf/v1/plugins/aggregators/
-  - /telegraf/v1/plugins/outputs/
-  - /telegraf/v1/plugins/inputs/
-  - /telegraf/v1/plugins/plugins-list/
-  - /telegraf/v1/plugins/aggregators/
-  - /telegraf/v1/plugins/inputs/
-  - /telegraf/v1/plugins/outputs/
-  - /telegraf/v1/plugins/processors/
-  - /telegraf/v1/plugins/plugins-list/
-  - /telegraf/v1/plugins/aggregators/
-  - /telegraf/v1/plugins/inputs/
-  - /telegraf/v1/plugins/outputs/
-  - /telegraf/v1/plugins/processors/
-  - /telegraf/v1/plugins/plugins-list/
-  - /telegraf/v1/plugins/aggregators/
-  - /telegraf/v1/plugins/inputs/
-  - /telegraf/v1/plugins/outputs/
-  - /telegraf/v1/plugins/processors/
-  - /telegraf/v1/plugins/plugins-list/
-  - /telegraf/v1/plugins/aggregators/
-  - /telegraf/v1/plugins/inputs/
-  - /telegraf/v1/plugins/outputs/
-  - /telegraf/v1/plugins/processors/
-  - /telegraf/v1/plugins/plugins-list/
   - /telegraf/v1/plugins/aggregators/
   - /telegraf/v1/plugins/inputs/
   - /telegraf/v1/plugins/outputs/
-  - /telegraf/v1/plugins/processors/
-  - /telegraf/v1/plugins/plugins-list/
-  - /telegraf/v1/plugins/aggregators/
-  - /telegraf/v1/plugins/inputs/
-  - /telegraf/v1/plugins/outputs/
-  - /telegraf/v1/plugins/processors/
+  - /telegraf/v1/plugins/plugin-list/
   - /telegraf/v1/plugins/plugins-list/
-  - /telegraf/v1/plugins/aggregators/
-  - /telegraf/v1/plugins/inputs/
-  - /telegraf/v1/plugins/outputs/
   - /telegraf/v1/plugins/processors/
```

Comment:
These must be relics from when we documented by minor version. A little find/replace action that never got fully cleaned up.

Reply:
Agreed - I figured it was cool to clean this long list up to just the unique values.

Reply:
Agreed - I figured it was cool to clean this long list up to just the unique values.